src/index.html

```diff
@@ -17,6 +17,90 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="wrapper">
+        <div class="navigation_logo">
+          <a
+            class="logo_link"
+            href="#"
+          >
```

Comment:
According to the checklist, you should avoid using spaces in `<a>` tag's `href` property. Anchor links should start with the `#` symbol.